pom.xml

```diff
@@ -149,7 +149,7 @@
         <version.com.jcraft.jzlib>1.1.1</version.com.jcraft.jzlib>
         <version.commons-io>2.5</version.commons-io>
         <version.commons-lang>2.6</version.commons-lang>
-        <version.io.undertow>2.0.30.Final</version.io.undertow>
+        <version.io.undertow>2.1.0.CR1-SNAPSHOT</version.io.undertow>
```

Comment:
@fl4via could you please update your PR with `2.1.0.Final` version please? thanks :)